onnxruntime/python/tools/transformers/float16.py

```diff
@@ -173,6 +173,7 @@ def convert_float_to_float16(
     op_block_list=None,
     node_block_list=None,
     force_fp16_initializers=False,
+    force_fp16_inputs=None,
```

Comment:
The purpose of force_fp16_inputs is to update `ALWAYS_FLOAT_INPUTS` dictionary. How about renaming it to force_fp32_inputs? When its value is None, we will use the default setting of `ALWAYS_FLOAT_INPUTS`. Otherwise, we will use the input dictionary as a replacement of `ALWAYS_FLOAT_INPUTS`.

Reply:
The problem with this is it requires the caller to know what `ALWAYS_FLOAT_INPUTS` contains in order to not accidentally move other ops to float16. For example, here I want to force GroupNorm inputs as float16 but I would have to pass `force_fp32_inputs={"Resize": [2]}` instead, which is counterintuitive. And as `ALWAYS_FLOAT_INPUTS` grows, the caller's code will need to be updated.

Reply:
I see. It's fine for me.